astropy/table/table_helpers.py

```diff
@@ -21,22 +21,21 @@ class TimingTables:
     are useful for timing and other API tests.
     """
 
-    def __init__(self, size=1000, masked=False):
+    def __init__(self, rng, size=1000, masked=False):
```

Comment:
Not sure I get what you mean here. The test seems to work as is and I don't see how `autouse=True` would help.

Reply:
OK. I had to use that in the past for some other fixture. If you don't need it, then nvm.

Reply:
This is not a test, so `rng` is not a `pytest` fixture here but a normal function parameter. The reason this backwards incompatible API change didn't cause any failures in CI is that the `TimingTables` class is not used anywhere. It would be better to remove it entirely instead of updating it.

Reply:
Ah, that explains it. Well, removing it now then.